src/config/dynamic-theme-fixes.config

```diff
@@ -14356,7 +14356,33 @@ CSS
 scribd.com
 
 INVERT
-.logo
+.wrapper__megamenu_top_bar .logo_button
+.scribd_blue_long_logo, .scribd_white_logo
+.auto__upload_index .icon_row .reason_cell .cell_icon
+.auto__static_about .about_content .fact, .auto__static_about .about_content .fun_facts
+.e584-3.x-image
+.auto__shared_fancy_header_header .logo_image
+._1gtMJh.f6Cwhy::before
+
+CSS
+.flat_btn.outline_btn.negate {
+    border-color: white !important;
+    color: rgb(232, 230, 227) !important; 
+}
+.jfk-bubble.gtx-bubble, .captcheck_answer_label > input + img, span#closed_text > img[src^="https://www.gstatic.com/images/branding/googlelogo"], span[data-href^="https://www.hcaptcha.com/"] > #icon, #bit-notification-bar-iframe, ::-webkit-calendar-picker-indicator, .wrapper__megamenu_top_bar .logo_button, .scribd_blue_long_logo, .scribd_white_logo, .auto__upload_index .icon_row .reason_cell .cell_icon, .auto__static_about .about_content .fact, .auto__static_about .about_content .fun_facts, .auto__static_about .about_content .fact, .auto__static_about .about_content .fun_facts
+{
+filter: invert(100%) hue-rotate(180deg) contrast(0%) brightness(170%)  !important;
+}
+.e584-3.x-image{
+filter: invert(100%) hue-rotate(180deg) contrast(0%) brightness(170%)  !important;
+}
+.auto__shared_fancy_header_header .logo_image
+{
+filter: invert(100%) hue-rotate(180deg) contrast(0%) brightness(170%)  !important;
+}
+.icon-ic_menuclosed:before{
+filter: invert(100%) hue-rotate(180deg) contrast(0%) brightness(170%)  !important;
+}
```

Comment:
Why isn't regular `INVERT` used for this?

Reply:
This is more about the brightness increase and not the invert, it makes the whole logo white instead of a weird pale blue that doesn't look that great on the black background. (basically removes the saturation)

Reply:
If I remove the invert(100%) part it still works the same way and turns it to an all white logo , which looks very sleek on the dark background.